schema/crdb/README.adoc

```diff
@@ -0,0 +1,60 @@
+:showtitle:
+:numbered:
+
+= Database Schemas
+
+This directory describes the schema(s) used by CockroachDB.
+
+We use the following conventions:
+
+* `schema/crdb/VERSION/up.sql`: The necessary idempotent migrations to transition from the
+  previous version of CockroachDB to this version.
+* `schema/crdb/VERSION/down.sql`: The necessary idempotent migrations to transition from
+  the current version of CockroachDB to the immediately preceding version.
+* `schema/crdb/dbinit.sql`: The necessary operations to create the latest version
+  of the schema. Should be equivalent to running all `up.sql` migrations, in-order.
```

Comment:
do we test that?

Reply:
We do, in `nexus/tests/integration_tests/schema.rs`.

- We rely on the `information_schema` virtual table in CockroachDB, and query for the state of tables, indices, columns, views, etc.
- We compare the version of the DB populated via `dbinit.sql` and the version of the DB populated via "all the up.sql files, in-order".
- We try to show a pretty diff if they're divergent, and we fail the test.